test/framework/shoot_utils.go

```
@@ -83,6 +90,38 @@ func (f *ShootFramework) GetValiLogs(ctx context.Context, valiLabels map[string]
 	return search, nil
 }
 
+func isValiAuthEnabled(ctx context.Context, log logr.Logger, selector labels.Selector, namespace string, c kubernetes.Interface) bool {
+	//Check if the auth is enabled in the configuration of the target
+	enabled, ok := authEnabledMap[selector.String()]
+	if ok {
+		return enabled
+	}
+
+	commandAuthEnabled := fmt.Sprintf("wget 'http://localhost:%d/config' -O-", valiPort)
+	r, err := PodExecByLabel(ctx, selector, valiLogging, commandAuthEnabled, namespace, c)
+	if err != nil {
+		log.Error(err, "Failed to check the vali config endpoint")
+		return false
```

Why the error is not returned?

---

thanks, the feedback is applied